core/src/main/java/org/apache/iceberg/hadoop/HadoopInputFile.java

```diff
@@ -30,6 +30,8 @@
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.SeekableInputStream;
 
+import static com.google.common.base.Preconditions.checkArgument;
```

Comment:
The project's style avoids static imports. Can you use `Preconditions.checkArgument` instead of just the method name where this is called?

Reply:
Will it be good to remove `com.google.common.base.Preconditions.*` exclusion from `AvoidStaticImport` rule in the project checkstyle?

Reply:
I am +1 on removing it from checkstyle in a separate commit as we technically allow such imports but try to avoid them.

Reply:
Yes. This should be caught by checkstyle.